pkg/route/api/v1/conversion.go

```diff
@@ -13,6 +13,9 @@ func init() {
 			obj.To.Kind = "Service"
 		},
 		func(obj *TLSConfig) {
+			if len(obj.Termination) == 0 {
+				obj.Termination = TLSTerminationEdge
```

Comment:
anything stored currently with an empty termination type should be considered a non-secure route.  If we try to default to edge termination we're going to force the users to use a wildcard or default, invalid certificate unless we have cert information in the old route.  

Previously, an empty termination type on the route was consider non-secure anyway.  The router should ignore the certs with an empty termination field unless something has changed.